apps/console/src/extensions/components/component-extensions/application/application-roles-tab.tsx

```diff
@@ -42,6 +46,20 @@ const ApplicationRolesTab: FunctionComponent<ApplicationRolesTabInterface> = (
 ): ReactElement => {
     const { application, onUpdate } = props;
 
+    if (applicationConfig?.advancedConfigurations?.useFluffyRoles) {
```

Comment:
If we add new ApplicationRolesTabInterface in features side, we can go without this config. We can keep this file as it is and add a new file in a directory like features/application/components/application-roles directory.

Reply:
Fixed with e3cb07a8648390285fe921cccc742cbba58d51c6